data/ayatana-indicator-display.desktop.in

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Type=Application
 Name=Ayatana Indicator Display
-Exec=@CMAKE_INSTALL_FULL_LIBEXECDIR@/ayatana-indicator-display-service
```

Comment:
Let's use @CMAKE_INSTALL_FULL_LIBEXECDIR@/ayatana-indicator-display/ayatana-indicator-display-service

Reply:
Why not `@CMAKE_INSTALL_FULL_PKGLIBEXECDIR@/ayatana-indicator-display-service`?

Reply:
For file configuration, we have dropped custom variables from most Ayatana projects, and are now using native CMake ones only. It improves readability (CMAKE_INSTALL_FULL_PKGLIBEXECDIR may also go in the future).

Reply:
Oh I thought this was a variable from CMake. I'll use your suggestion then